tool/tsh/tsh.go

```diff
@@ -702,8 +704,9 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 	proxyDB := proxy.Command("db", "Start local TLS proxy for database connections when using Teleport in single-port mode")
 	proxyDB.Arg("db", "The name of the database to start local proxy for").Required().StringVar(&cf.DatabaseService)
 	proxyDB.Flag("port", "Specifies the source port used by proxy db listener").Short('p').StringVar(&cf.LocalProxyPort)
-	proxyDB.Flag("cert-file", "Certificate file for proxy client TLS configuration").StringVar(&cf.LocalProxyCertFile)
-	proxyDB.Flag("key-file", "Key file for proxy client TLS configuration").StringVar(&cf.LocalProxyKeyFile)
+	// --cert-file and --key-file are deprecated in favor of --tunnel flag.
+	proxyDB.Flag("cert-file", "Certificate file for proxy client TLS configuration").Hidden().StringVar(&cf.LocalProxyCertFile)
+	proxyDB.Flag("key-file", "Key file for proxy client TLS configuration").Hidden().StringVar(&cf.LocalProxyKeyFile)
```

Comment:
I think that the cert-file and key-file are still useful in case when the db certs was created by `tctl auth sign` like:  `tctl auth sign --format tls -o db_credentials --db-name db-1 --user alice --ttl 1h` and the local proxy is needed to connect to Teleport Proxy. 

If we remove these flags in some cases a use with `tctl auth sign` won't be able to use database access where tsh local proxy is required.

Reply:
After having a second look at code the `cert-file` and `"key-file` flags are related to SQLServer access before `--tunnel` flag  was added and we never had support for `tctl auth sign` exporetd  certs in local proxy.

Reply:
> After having a second look at code the `cert-file` and `"key-file` flags are related to SQLServer access before `--tunnel` flag was added and we never had support for `tctl auth sign` exporetd certs in local proxy.

so then we can hide and deprecate these flags?

Reply:
Yes. Since these flags were only used in use case os MSSQL protocol after `--tunnel` flag was introduced and we never handled the exported `tctl auth sign` `certs in tsh local proxy` we can hide these flags.